sw/device/lib/crypto/impl/aes_gcm/ghash.c

```diff
@@ -275,6 +276,13 @@ static ghash_block_t galois_mul_state_key(ghash_block_t state,
  * @param block Block to incorporate.
  */
 static void ghash_process_block(ghash_context_t *ctx, ghash_block_t *block) {
+  // ibex_clear_rf() below cannot clear x8 (frame pointer). To avoid that
+  // x8 is not used for secret data, create a variable-length array (VLA).
+  // This forces the compiler to use x8 instead of the stack pointer,
+  // avoiding that x8 can be used for secret data.
+  uint32_t dummy[launderw(1)][4];
+  barrierw((uintptr_t)&dummy);
```

Comment:
Have you tried to put this in `ibex_clear_rf()` instead?

Also, I think we can use `launderw(0)` as we allow zero-sized array extension. @luismarques